seed/data_importer/match.py

```diff
@@ -256,6 +258,10 @@ def inclusive_match_and_merge(unmatched_state_ids, org, StateClass):
     # Collapse groups of matches found in the previous step into 1 -State per group
     merges_within_file = 0
     priorities = Column.retrieve_priorities(org)
+    total_ids = len(matched_id_groups)
```

Comment:
I like what you did here. It seems like we're stepping the progress data each we've gone through `batch` number of properties. A pretty traditional way of doing this is with the modulo operator. And we can use `enumerate` to also get the "index" we're working with. For example:
```python
batch_size = 100
for idx, ids in enumerate(matched_id_groups):
    if idx % batch_size == 0:
        print(f'I did another {batch_size} items!')
```

Reply:
That's so clean!